modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -7,19 +7,49 @@ Before deploying a Mule application to Anypoint Runtime Fabric, determine the co
 
 When a Mule application is deployed to Runtime Fabric, the application is deployed with its own Mule runtime engine (Mule). The number of replicas, or instances of that application and runtime, are also specified. The resources available for each replica are determined by the values you set when deploying an application.
 
+The performance information provided in this topic is based on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. M4 instances feature a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2. They run at a base clock rate of 2.4 GHz, and can go as high as 3.0 GHz with Intel Turbo Boost. The load generator used was hosted on a separate AWS instance in the same region.
+
 You can allocate the following resources when deploying an application:
 
 * vCPU cores
+** `Reserved vCPU` - the amount of vCPU guaranteed to the application and reserved for its use.
```

Comment:
GLOBALLY break list items and steps following the first sentence or fragment of text, to enhance scannability (the purpose of the format).
You can also use a definition list; see the style ref.

Reply:
```suggestion
** `Reserved vCPU`
The amount of vCPU guaranteed to the application and reserved for its use
```

Reply:
Updates applied as needed.